release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -70,9 +70,43 @@ This release adds improvements related to the following components and concepts.
 === Registry
 
 
-[id="ocp-4-8-operators"]
+[id="ocp-4-8-olm"]
 === Operator lifecycle
 
+[id="ocp-4-7-admin-error-reporting"]
+==== Enhanced error reporting for administrators
+
+A cluster administrator using Operator Lifecycle Manager (OLM) to install an Operator might encounter several error conditions that are related either to the current API or related to low-level APIs. Previously, there was little insight into why OLM could not fulfill a request to install or update an Operator. These issues could range from trivial things like typos in object properties or missing RBAC to medium complexity where items cannot be loaded from the catalog because there are metadata parsing issues.
+
+Because administrators should not require understanding of the interaction process between the various low-level APIs or access to the OLM pod logs to successfully debug such issues, {product-title} 4.8 introduces the following enhancements in OLM to provide administrators with more comprehensible error reporting and messages:
+
+[id="ocp-4-7-installplan-errors"]
+Retrying install plans::
+Install plans, defined by an `InstallPlan` object, can encounter transient errors, for example, due to API server availability or conflicts with other writers. Previously, these errors would result in the termination of partially-applied install plans that required manual cleanup. With this enhancement, the Catalog Operator now retries errors during install plan execution for up to one minute. The new `.status.message` field provides a human-readable indication when retries are occurring.
+
+[id="ocp-4-7-operatorgroup-errors"]
+Indicating invalid Operator groups::
+Creating a subscription in a namespace with no Operator groups or multiple Operator groups would previously result in a stalled Operator installation with an install plan that stays in `phase=Installing` forever. With this enhancement, the install plan immediately transitions to `phase=Failed` so that the administrator can correct the invalid Operator group and try again.
+
+[id="ocp-4-7-candidate-operator-errors"]
+Specific reporting when no candidate Operators found::
+`ResolutionFailed` events, which are created when resolving dependencies in a namespace fails, now provide more specific text when the namespace contains a subscription that references a package or channel that does not exist in the referenced catalog source. Previously, this message was generic:
++
+[source,terminal]
+----
+no candidate operators found matching the spec of subscription '<name>'
+----
++
+However, with this enhancement, the message is more specific:
++
+[source,terminal]
```

Comment:
It makes me sad how these highlight all funky via `terminal` and I now also wish we had allowed `text` for random output/messages. :|

Reply:
Hmm just noticed it's only funky highlighting in the local Asciibinder build, but the Netlify build is fine... :shrug: